pkg/coordinator/coordinator.go

```diff
@@ -287,15 +314,17 @@ func (b idBlocker) shouldUpdate(val idBlocker) (bool, error) {
 	// Now the checkBlockNumber should be same
 
 	// If b has an IndefiniteBlockingKey, then update
-	if b.TransmitBlockNumber.String() == IndefiniteBlockingKey.String() {
+	if string(b.TransmitBlockNumber) == string(IndefiniteBlockingKey) {
```

Comment:
Because the type doesn't have a `String()` method anymore and putting one on `[]byte` doesn't really make much sense. We *could*, but not sure what value it provides anymore.

Reply:
So I think a `String()` function makes sense, since it gives us that control of how to represent a type in string form, if we ever graduate beyond using byte arrays under the hood? But right now it's cool to keep it this way! 👍